config/wrapper/pci_input.conf

```diff
@@ -23,6 +60,38 @@ pci_device = functions:0
 pci_devices = functions:all
 module = driver
 
+[io_iscsi_fvt]
+disk = disks:0
+htx_disks = disks:all
+disks = disks:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver
+
+[io_lsi3008_fvt]
+disk = disks:0
+htx_disks = disks:all
+disks = disks:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver
+
+[io_lsi3961_fvt]
+disk = disks:0
+htx_disks = disks:all
+disks = disks:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver
```

Comment:
is this reflection of bucket has new test case added or deletion

Reply:
Enabling the Dynamic input selection code to handle the new IO technology.